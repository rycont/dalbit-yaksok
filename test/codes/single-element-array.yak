R = [10]
R 보여주기

S = ["hello"]
S 보여주기

T = [3.14]
T 보여주기