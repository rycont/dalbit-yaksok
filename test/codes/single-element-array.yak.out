[10]
[hello]
[3.14]
